api/lib/mno_enterprise/concerns/controllers/pages_controller.rb

```diff
@@ -65,7 +65,9 @@ def terms
     ts = MnoEnterprise::App.order_by("updated_at.desc").first.try(:updated_at)
     @apps = if ts
       Rails.cache.fetch(['pages/terms/app-list', ts]) do
-        MnoEnterprise::App.order_by("name.ac").reject{|i| i.terms_url.blank?}
+        # Temp solution as translated fields can not be filtered or sorted
+        # MnoEnterprise::App.order_by("name.ac").reject{|i| i.terms_url.blank?}
+        MnoEnterprise::App.all.reject { |i| i.terms_url.blank? }.sort! { |a,b| a.name.downcase <=> b.name.downcase }
```

Comment:
why not `sort_by{|h|h.name.downcase}`

Reply:
Sure, like yours more, 0 advantages of my approach compare to yours.
Updated.